### PR TITLE
standard-config: loosen bounds on `l2oo.submissionInterval`

### PR DESCRIPTION
**Pull request by geoknee**
Output frequency can be 24 hours or less: https://specs.optimism.io/protocol/configurability.html#policy-parameters

### CODE REVIEW

File: validation/l2oo_test.go [hunk 1]
```diff
@@ -19,33 +19,23 @@ import (
...
-		8866:      true, // mainnet/superlumio Incorrect submissionInterval
-	}
-
-	checkEquality := func(a, b *big.Int) func() bool {
-		return (func() bool { return (a.Cmp(b) == 0) })
-	}
-
-	incorrectMsg := func(name string, want, got *big.Int) string {
-		return fmt.Sprintf("Incorrect %s, wanted %d got %d", name, want, got)
+		999999999: true, // sepolia/zora  Incorrect finalizationPeriodSeconds, 604800 is not within bounds [12 12]
```
Comment (geoknee) [inline validation/l2oo_test.go:30]: `finalizationPeriodSeconds` on Sepolia is an obvious place to look next. I suspect being strict about `12` is not what we want to do.

File: validation/l2oo_test.go [hunk 2]
```diff
@@ -19,33 +19,23 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
+type L2OOParams struct {
+	SubmissionInterval        *big.Int
+	L2BlockTime               *big.Int
+	FinalizationPeriodSeconds *big.Int
```
Comment (blmalone) [inline validation/l2oo_test.go:25]: Do we want to change the name of this variable to be challenge period? Maybe not as I know the on-chain function is `finalizationPeriodSeconds()`

**Review by blmalone — APPROVED** [review]
LGTM 👍🏻

File: validation/l2oo_test.go (hunk 2 above)
Comment (geoknee) [reply validation/l2oo_test.go:25]: (So keep both names around, one on the data scraping side, one on the spec side).

**Review by bitwiseguy — APPROVED** [review]
Lgtm

File: validation/l2oo_test.go (hunk 1 above)
Comment (bitwiseguy) [reply validation/l2oo_test.go:30]: Where is the `604800` value coming from?

File: validation/l2oo_test.go (hunk 2 above)
Comment (mds1) [reply validation/l2oo_test.go:25]: Just unresolved for visibility: It's a bit unfortunate there's two names, I'm indifferent to the name used as long as it's clear they are the same. Maybe just a comment in the toml file is sufficient:
```toml
challenge_period_seconds = [12, 12] # i.e. finalization period
```

Comment (geoknee) [reply validation/l2oo_test.go:25]: [9cb8ac3](https://github.com/ethereum-optimism/superchain-registry/pull/208/commits/9cb8ac3b779d189455dd9c46ad01439cf1edcc6a)

File: validation/l2oo_test.go (hunk 1 above)
Comment (geoknee) [reply validation/l2oo_test.go:30]: 604800 is being used on sepolia/zora. It's one week's worth of seconds.

**Pull request merged**